client/src/mergeUtils.js

```diff
@@ -275,11 +275,14 @@ export function getActivationButton(isActive, onClickAction, instanceName) {
       intent={isActive ? Intent.DANGER : Intent.SUCCESS}
     >
       <Button
-        icon={isActive ? IconNames.STOP : IconNames.PLAY}
-        outlined
-        intent={isActive ? Intent.DANGER : Intent.SUCCESS}
+        variant={isActive ? "outline-danger" : "outline-success"}
         onClick={onClickAction}
-      />
+      >
+        <Icon
+          icon={isActive ? IconNames.STOP : IconNames.PLAY}
+          intent={isActive ? Intent.DANGER : Intent.SUCCESS}
```

Comment:
Property intent should be removed because when button is hovered over, red icon on red background is barely visible.